pkg/common/crypto_test.go

```diff
@@ -0,0 +1,24 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type CryptoTestSuite struct {
```

Comment:
We don't need a testify suite for this given that we're just doing functional tests for crypto.go. Suites is used more more advanced structs / where we need construction/destruction of structs, and other misc things in a test.

Reply:
Let's also write some TDD's for Sha3_256HashHex and GetAddressFromPublicKey, no need for Sha3_256Hash since it returns a byte buffer.